components/Stands/LivePlayer.tsx

```diff
@@ -1,11 +1,11 @@
 import React from 'react';
 import styled from 'styled-components';
 
-const LiveContainer = styled.div`
+const LiveContainer = styled.div<{ frontPage?: boolean }>`
```

Comment:
I can't see this used anywhere?

Reply:
Good point, I changed the Container to be independent of which page it's on - it can be deleted